### DISCUSSION TITLE
Teacher verification

### DISCUSSION THREAD

**Discussion by jtocch** [community/community#38528] [GitHub Education]
I'm having issues processing my teacher verification application... I have followed the guidelines and I'm unable to proceed further. Any advice would be great.

**Comment by samus-aran** [comment]
Hi @jtocch  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by jtocch** [comment]
Hi Ashley,

Thanks for your email.

Error messages have varied. The error message that I’m getting now relates to campus location in my country.

***@***.***

If you can advise of my next steps that would be greatly appreciated.

Kind Regards

Jenny


[cid:ECBanner_c15c5243-2141-47bf-bf21-318439f08451.JPG]<http://web.ecmelb.catholic.edu.au>      Jenny Tocchetto
Teacher
***@***.***
T 03 8325 5100
St Paul's Campus
423 Blackshaws Road
Altona North    F 03 9314 2475
PO Box 5
Altona North
Vic 3025

This Email and any attachments may be confidential and, if you are not the intended recipient, you must not disclose or use the information in this mail. If received in error, please notify us immediately and delete the Email and all copies. The school does not guarantee that this Email is virus or error free. The attached files are provided and may only be used on the basis that the user assumes all responsibility for any loss, damage or consequence resulting directly or indirectly from the use of the attached files, whether caused by the negligence of the sender or not. The content and opinions in this Email are not necessarily those of the school.
